src/main.ts

```diff
@@ -15,23 +15,37 @@ export class Main extends Command {
   }
 
   async run() {
+    if (this.argv.length === 0) return this._runDefaultCommand()
     let [id, ...argv] = this.argv
     this.parse({strict: false, '--': false, ...this.ctor as any})
     if (!this.config.findCommand(id)) {
       let topic = this.config.findTopic(id)
       if (topic) return this._help()
+      return this._runDefaultCommand()
```

Comment:
Nice PR! We shouldn't intervene here though, rather, let the lifecycle events continue to the command_not_found hook.

Reply:
I guess I wanted to make the default command a native feature of the library, instead of handling it as a hook. One reason for doing this is to circumvent the built in behavior to show the help page when no arguments are passed to the CLI.